blueprint-test-utils/src/tangle/mod.rs

```diff
@@ -0,0 +1,165 @@
+use crate::tangle::node::{Error, SubstrateNode, TANGLE_NODE_ENV};
+
+pub mod node;
+pub mod transactions;
+
+/// Run a Tangle node with the default settings.
+/// The node will shut down when the returned handle is dropped.
+pub fn run() -> Result<SubstrateNode, Error> {
+    let tangle_from_env = std::env::var(TANGLE_NODE_ENV).unwrap_or_else(|_| "tangle".to_string());
+    let builder = SubstrateNode::builder()
+        .binary_paths([
+            "../tangle/target/release/tangle",
+            "../../tangle/target/release/tangle",
+            &tangle_from_env,
+        ])
+        .arg("validator")
+        .arg_val("rpc-cors", "all")
+        .arg_val("rpc-methods", "unsafe")
+        .arg("rpc-external")
+        .arg_val("sealing", "manual")
+        .clone();
+    builder.spawn()
+}
+
+#[macro_export]
+macro_rules! generic_test_blueprint {
```

Comment:
What's the difference between this macro and test tangle blueprint macro? The naming is not very clear what generic means here.